helm-projectile.el

```diff
@@ -668,7 +668,11 @@ If it is nil, or ack/ack-grep not found then use default grep command."
                       (assoc-default 'follow helm-source-grep)))
          (helm-grep-in-recurse t)
          (helm-grep-ignored-files (-union (projectile-ignored-files-rel)  grep-find-ignored-files))
-         (helm-grep-ignored-directories (-union (projectile-ignored-directories-rel) grep-find-ignored-directories))
+         (helm-grep-ignored-directories
+          (-union (--map (and (string-suffix-p "/" it)
```

Comment:
You should just map `directory-file-name` over the list. No need for the check and the custom string manipulation.